### DISCUSSION TITLE
A problem about marking text

### DISCUSSION THREAD

**Discussion by zhubib1234** [javalent/obsidian-leaflet#81] [Q&A]
Is it difficult to mark text directly on the map?

**Comment by valentine195** [comment]
Hmm, let me think about it. This could be difficult - does the text need to scale on zoom? etc...

**Reply by zhubib1234** [reply]
The text does not need to be scaled. thanks!

**Reply by Moonbase59** [reply]
Could maybe use part of the mapoverlay code: Specify _where_ and put some text instead of the circle overlay? Could reuse color, location and description.
